ndmg/register.py

```diff
@@ -266,6 +267,17 @@ def gen_tissue(self):
         print("Extracting brain from raw T1w image...")
         reg_utils.t1w_skullstrip(self.t1w, self.t1w_brain, self.skull)
 
+        #  QA part of skull strip
+        last_folder = self.namer.dirs["qa"]["base"]
+        QA_skullstrip_path = last_folder + '/skull_strip'
```

Comment:
`last_folder` is only used once, change to:
`qa_skullstrip_path = f'{self.namer.dirs["qa"]["base"]}/skullstrip' `

Reply:
fixed it in latest version

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834670-ee7c2700-1dc8-11ea-9e2c-45f1320a4cb1.png)